docs-source/content/faq/external-clients.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "External WebLogic Clients"
+date: 2019-11-21T21:23:03Z
+draft: false
+weight: 6
+---
+
+### Approaches
+
+There are two supported approaches for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster: [Load Balancer Tunneling](#load-balancer-tunneling) and [Kubernetes Node Ports](#kubernetes-node-ports).
+
+> __NOTE:__ This FAQ is for remote EJB and JMS clients - not JTA clients. The operator does not currently support external WebLogic JTA access to a WebLogic cluster, as (a) external JTA access requires each server in the cluster to be individually addressable by the client, but this conflicts with (b) the current operator requirement that a network channel in a cluster have the same port across all servers in the cluster.
+
+#### Load Balancer Tunneling
+
+The Load Balancer Tunneling approach for giving external WebLogic EJB or JMS clients access to a Kubernetes hosted WebLogic cluster involves configuring a network channel on the desired WebLogic cluster that accepts `T3` protocol traffic that's tunnelled over HTTP, deploying a load balancer that redirects external HTTP network traffic to the desired WebLogic network channel, and ensuring tha EJB and JMS clients specify a URL that resolves the load balancer's network address.  
+
+Here are the specific steps:
+
+- Configure a custom channel for the T3 protocol in WebLogic that (a) enables HTTP Tunneling, and (b) specifies an external address and port that correspond to the address and port remote clients will use access the load balancer.  See  [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel) for samples and details.
+
+- Setup a load balancer that redirects HTTP traffic to the custom channel. For a discussion of load balancers see [Ingress]({{<relref "/userguide/managing-domains/ingress/_index.md">}}). If you're also using OKE/OCI to host your Kubernetes cluster, also see [Using an OCI Load Balancer]({{<relref "weblogic-kubernetes-operator/faq/oci-lb/_index.md">}}).
```

Comment:
Setup -> Set up   (set up (v), setup (n) (adj))
For a discussion of load balancers see -> For a discussion of load balancers, (comma) see